From 42abfa732b8ed8d2ff221facf6a8c122cb212088 Mon Sep 17 00:00:00 2001
From: Chen Liangjun <b36089@freescale.com>
Date: Wed, 11 Jan 2012 15:03:22 +0800
Subject: [PATCH 268/505] ENGR00171742 ASRC: workround of sdma buf limit bug

The limit of sdma buffer size is 64k, and the input buffer size is
10k. So the output sample rate must be less than 6 times of input
sample rate.

workround: reduce the input buffer size to 4k.

Signed-off-by: Chen Liangjun <b36089@freescale.com>
---
 test/mxc_asrc_test/mxc_asrc_test.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/mxc_asrc_test/mxc_asrc_test.c b/test/mxc_asrc_test/mxc_asrc_test.c
index ceda7e5..5beb6e7 100644
--- a/test/mxc_asrc_test/mxc_asrc_test.c
+++ b/test/mxc_asrc_test/mxc_asrc_test.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2008-2011 Freescale Semiconductor, Inc. All rights reserved.
+ * Copyright 2008-2012 Freescale Semiconductor, Inc. All rights reserved.
  */
 
 /*
@@ -29,7 +29,7 @@
 #include <sys/time.h>
 #include <linux/mxc_asrc.h>
 
-#define DMA_BUF_SIZE 10240
+#define DMA_BUF_SIZE 4096
 /*
  * From 38 kernel, asrc driver only supports one pair of buffer
  * convertion per time
-- 
1.8.0

